content/docs/14-troubleshooting/30-pcg.md

```diff
@@ -88,37 +103,220 @@ When the PCG installer instance is powered on, it goes through a bootstrap proce
+
+
+9. Share the logs with Spectro Cloud's support team at [support@spectrocloud.com](mailto:support@spectrocloud.com).
+<br />
+
+
+# Scenario - No Progress After Creating the Container Manager
+Suppose the PCG events in Palette display no progress after the specific event, `Created container manager`. 
+
+The issue can occur when the gateway installer VM fails to connect to the Spectro Cloud API, [https://api.spectrocloud.com](https://api.spectrocloud.com), to download the installation artifacts. Another potential reason is that the gateway installer may not have the required permissions to store the installation artifacts in the **spectro-templates** folder. The installer downloads the images for the worker nodes and stores them in the **spectro-templates** folder during the cluster provisioning.
+<br />
 
 ## Debug Steps
+<br />
+
+1. Check the network connectivity from the gateway installer VM to the Spectro Cloud API. Ensure that no network issues or firewall restrictions are blocking the connection.
 
+
+2. Ensure you have the necessary write permissions for the **spectro-templates** folder. 
 <br />
 
-1. Review the cluster details page, which can be accessed by clicking anywhere on the gateway widget. The details page contains information about every orchestration step, including the current task. Any intermittent errors will be displayed on the cluster details page next to the relevant orchestration task. 
+# Scenario - Failed to Deploy Image
 
+Suppose the PCG events in Palette display the `Failed to deploy image: Failed to create govomiClient` error. 
 
-2. Click on the **Events** tab to review lower-level operations being performed for the various orchestration steps.
+The error can occur if there is a preceding "https://" or "http://" string in the vCenter server URL or the gateway installer VM lacks internet connectivity.
+<br />
 
+## Debug Steps
+<br />
+
+1. Check the data center server URL in the gateway cluster configurations. Ensure the URL does not include the preceding "http://" or "https://" string. 
+
+
+2. Check the internet connectivity from within the gateway installer VM. Ensure that no network issues or firewall restrictions are blocking the connection.
+
+
+3. Suppose the gateway installer VM does not have outbound internet access. In that case, you can either provision the gateway installer VM in a network that supports outbound connectivity or fix the firewall rules of the existing network.
+<br />
+
+
+# Scenario - No Route to the Kube API Server
+
+While deploying the cloud gateway cluster, the Palette events display the `No route to host` error. 
+
+The error indicates an issue with the PCG cluster nodes attempting to connect to the Kube API server. This issue can occur due to improper networking configuration or an error in the cloud-init process. 
+<br />
+
+## Debug Steps
+<br />
+
+1. Ensure you provision the gateway cluster VM in a network supporting outbound connectivity. 
```

Comment:
What is the outbound connection targeting? Palette?  I'm not sure what the outbound connection is for.